src/main/scala/uk/gov/hmrc/hmrcfrontend/views/config/HmrcPageHeading.scala

```diff
@@ -17,7 +17,7 @@
 package uk.gov.hmrc.hmrcfrontend.views.config
 
 import play.twirl.api.{Html, HtmlFormat}
-import uk.gov.hmrc.govukfrontend.views.Utils.toClasses
+import uk.gov.hmrc.supportfrontend.views.Utils.toClasses
```

Comment:
Presumably slight risk teams could be using Utils directly?

Reply:
Hmm, good point. We could make this non-breaking by having this code in a trait in a trait and preserving the original companion objects. Shall I make this change?

Reply:
With a deprecation notice?